Presentation.Web/app/components/org/user/org-user.controller.ts

```diff
@@ -419,6 +446,13 @@
             this.mainGridOptions = mainGridOptions;
         }
 
+        private readonly excelConfig: Models.IExcelConfig = {
```

Comment:
is this used for anything other than being passed as an object... my point is.. does it have to be an object which lives on the class or could it just be a `{}` passed to the call?

Reply:
this one is used to store the visibility config